### PR TITLE
F/image upload changes

### PR DESCRIPTION
**Pull request by jugarrit**
This includes the removal of Eclipse instructions, but we can always take that commit out.

@jpjoyal 
@dannytranlx 
@Mario54 
@alavers

### CODE REVIEW

File: source/includes/_configuring_push_android.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ Be sure to replace `<your_application_package>` with your application package!
 1. Add the dependency to your project's top-level `build.gradle`:
 
     ```
-    classpath 'com.google.gms:google-services:1.3.0-beta1'
+    classpath 'com.google.gms:google-services:2.0.0-alpha6'
```
Comment (mspensieri) [inline source/includes/_configuring_push_android.md]: switch from beta to alpha... do these guys ever do legitimate releases?

**Comment by mspensieri** [comment]
👍

**Comment by mspensieri** [comment]
Do the instructions in the web app need to be updated as well?

**Comment by jugarrit** [comment]
Just checked the instructions in the web app, they don't mention Eclipse at all.

**Comment by jugarrit** [comment]
Although we'll need to change the readme in the public repo

**Comment by dannytranlx** [comment]
👍

**Pull request merged**